fawltydeps/packages.py

```diff
@@ -13,6 +13,8 @@
 from pathlib import Path
 from typing import Dict, Iterable, Iterator, List, Optional, Set
 
+import pytest
```

Comment:
This is the core of what I want to avoid, since I don't want to have `pytest` become a _main_ dependency of FawltyDeps.